platform/darwin/src/MGLStyleValue.h

```diff
@@ -75,9 +75,9 @@ typedef NS_ENUM(NSUInteger, MGLInterpolationMode) {
      */
     MGLInterpolationModeCategorical,
     /**
-     Values between two stops are not interpolated. Instead, values are set to their
-     input value. Use identity interpolation mode to show attribute values that can be
-     used as style values.
+     Values between two stops are not interpolated. Instead, style values are set
+     to the attribute value for a feature. Use identity interpolation mode to show 
```

Comment:
Maybe we should say “match”, “correspond to”, “determined by” instead of “set to”, so that the developer doesn’t misinterpret this sentence to mean that they have to set a value explicitly?

> Instead, for any given feature, the style value matches a value in that feature’s `attributes` dictionary.